scripts/qesap/test/unit/test_qesap_lib_cmd_ansible.py

```diff
@@ -0,0 +1,37 @@
+from unittest import mock
+import logging
+import os
+
+
+from lib.cmds import export_ansible_output
+
+log = logging.getLogger(__name__)
+
+
+def test_export_ansible_output():
```

Comment:
Here you can see the execution of the single new test in this file `[test_qesap_lib_cmd_ansible.py](https://github.com/SUSE/qe-sap-deployment/pull/234/files#diff-64bd1de2060c01d07e450f4bb3a2ce75e4827907a733d818310885ab8ccced55)`  --> log https://github.com/SUSE/qe-sap-deployment/actions/runs/9481063837/job/26123325056#step:7:33

Reply:
And here an execution example with verbose mode on 
https://github.com/SUSE/qe-sap-deployment/actions/runs/9481063837/job/26123325056#step:8:838